src/components/popup/style.css

```diff
@@ -0,0 +1,96 @@
+.popupWrapper {
+  display: none;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  z-index: 1100;
+  background-color: rgba(0, 0, 0, 0.276);
+}
+
+.popupWrapper .popup {
+  width: 60%;
+  padding: 45px;
+  background-color: white;
+  border-radius: 12px;
+  position: relative;
+  display: flex;
+  gap: 30px;
+}
+
+.popupWrapper .popup .content-description {
```

Comment:
you can wrap css
.popup {
  ...common popup css
  .content {
    ...content css
  {
{